server/README.md

```diff
@@ -0,0 +1,80 @@
+# API
+<!-- Endpoints are meant to be preceded by /api (/operations == /api/operations)-->
+## Bookings
+- GET `/bookings`
+  - Param: Time frame (start date and end date)
+  - Response: List of all bookings
+  ##### NOTE: This endpoint is intended to be used for statistics 
+
+## Lectures
+- GET `/lectures`
+  - Param: NONE
+  - Response: List of all lectures
+- GET `/lectures/{lecture_id}`
+  - Param: lecture_id
+  - Response: Lecture object for the given id
+- GET `/lectures/{lecture_id}/bookings`
+  - Param: lecture_id
+  - Response: List of all bookings for a given lecture
+- PATCH: `/lectures/{lecture_id}`
+  - Param lecture_id
+  - Body: {virtual: true/false}
+  - Response: NONE
+- POST: `/lectures`
+  - Body: Lecture object (used to add a lecture)
+  - Response: NONE
+- DELETE: `/lectures/{lecture_id}`
+  - Param: lecture_id
+  - Response: NONE
+  
+## Courses
+- GET `/courses`
```

Comment:
Should the output of this call depend on the logged in user too?

Reply:
Ah okay I just reached the `students` endpoint, I get it. It makes sense, you can resolve this discussion.